content/docs/main/integrations/service-entry.md

```diff
@@ -0,0 +1,339 @@
+3. Follow the [Istio documentation](https://istio.io/latest/docs/ambient/getting-started/) to install Istio and set up [Ambient Mesh](https://kgateway.dev/docs/main/integrations/istio/ambient/ambient-ingress/) in your cluster. 
+
+## Targeting ServiceEntry Backends with HTTPRoute and TrafficPolicies
+
+1. Create a Testing Namespace `gwtest` to deploy our Gateways, HTTPRoutes and ServiceEntries.
+
+```yaml
+kubectl create namespace gwtest
+```
+
+2. The `Gateway` resource, listens for HTTP traffic on port `8080`. The `HTTPRoute`, then matches incoming requests for the hostname and directs them to a backend specified as an Istio `ServiceEntry`. This separation of concerns means the ingress logic remains constant, while the `ServiceEntry` itself dictates the backend's discovery and resolution strategy.
+
+```yaml
+kubectl apply -f - <<EOF
+kind: Gateway
+apiVersion: gateway.networking.k8s.io/v1
+metadata:
+  name: http-gw-for-test
+  namespace: gwtest
+spec:
+  gatewayClassName: kgateway
+  listeners:
+  - protocol: HTTP
+    port: 8080
+    name: http
+    allowedRoutes:
+      namespaces:
+        from: All
+---
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: HTTPRoute
+metadata:
+  name: route-to-upstream
+  namespace: gwtest
+spec:
+  parentRefs:
+  - name: http-gw-for-test
+  hostnames:
+  - "se.example.com"
+  rules:
+  - backendRefs:
+    - name: example-se
+      port: 80
+      kind: ServiceEntry
+      group: networking.istio.io
+EOF
+```
+
+3. A TrafficPolicy allows you to modify the behavior of traffic on a specific route, adding advanced rules like timeouts, retries, and request transformations. This example applies a 5-second timeout and 3 retries to any traffic that goes through the HTTPRoute you have already defined
+
+```yaml
+kubectl apply -f - <<EOF
+apiVersion: gateway.networking.k8s.io/v1alpha2
+kind: TrafficPolicy
+metadata:
+  name: external-service-policy
+  namespace: gwtest
+spec:
+  targetRef:
+    kind: HTTPRoute
+    name: route-to-upstream
+  http:
+    timeout: 5s
+    retry:
+      retries: 3
+      perTryTimeout: 2s
+EOF
+```
+
+## Full Code Examples & Use Cases
+
+Building on the common kgateway and HTTPRoute configuration, these examples demonstrate how different `ServiceEntry` types are implemented to integrate various external service scenarios.
+
+### 1\. Static Endpoints
+
+This configuration is the most straightforward, designed for external services with fixed, known network addresses. It's an excellent choice for integrating a legacy database, an on-premises application with a stable IP, or for controlled test environments.
+
+The `ServiceEntry` explicitly lists the IP addresses of the external service in the `endpoints` array. With `resolution: STATIC`, Istio directly distributes traffic among these predefined addresses.
+
+```yaml
+kubectl apply -f - <<EOF
```

Comment:
So for the static IP example, you still need to reference an example endpoint. If you're including the httpbin app, maybe include a step to get the IP of the httpbin app and use that instead so the validation step works.